mod-notes/mod-notes.postman_collection.json

```diff
@@ -8215,161 +8214,1450 @@
 			]
 		},
 		{
-			"name": "teardown notes",
+			"name": "note links",
 			"item": [
 				{
-					"name": "DELETE testing note",
-					"event": [
+					"name": "Positive",
+					"item": [
 						{
-							"listen": "prerequest",
-							"script": {
-								"id": "28b5c6af-1d14-4c12-bf64-4ad6b13b3c1e",
-								"exec": [
-									""
+							"name": "/note-links/type/{type}/id/{id}/ - 204 assign link to non existing records",
+							"event": [
```

Comment:
Shouldn't we be seeing an error if attempting to assign links to a non existing note? Question on implementation itself

Reply:
Yeah, but it's not clear what should be returned in this case, because request contains an array of notes, so some notes can be updated, and some other notes can be not update (because they don't exist, or because they already had this link),
it's possible to return a response with a list of ok/error responses

Reply:
ok -- thanks for details @andrii-paias -- may need to revisit once functionality is being used from UI. But API tests reflect current functionality